### DISCUSSION TITLE
Wiki Structure

### DISCUSSION THREAD

**Discussion by JaJako** [Oeffis/Oeffis#15] [General]
How to structure the wiki?

As I found out as a previous non-GitHub user there is no option to group wiki pages or arrange them in a tree-like structure. Just a large unstructured set of pages or multiple very long pages would not be my first choice.

Part of #36

**Comment by JaJako** [comment]
As I found out you can add a custom sidebar with a nested list of all the pages structured according to your taste. (I added such a sidebar test-wise to our wiki). But such a solution we would have to maintain with every new (or renamed?) side.

There seems to be some solutions to automatically generated such a sidebar depending on all your wiki pages, for example:
- https://github.com/adriantanasa/github-wiki-sidebar/wiki
- https://github.com/gollum/gollum

**Reply by timvahlbrock** [reply]
Automatic generation sounds good 👍 

maybe this can somehow be automatically done before committing or in pipeline? But I won't spend too much time on that.

**Comment by JaJako** [comment]
Closed - Wiki has been tidied up by #36 . Further automation (if desired) is covered by #105 .
